assets/model_monitoring/components/src/shared_utilities/store_url.py

```diff
@@ -225,15 +227,21 @@ def any_files(file_names: list, pattern):
         container_client = container_client or self.get_container_client()
         if not container_client:  # local
             return any(glob.iglob(full_path_pattern))
-        if isinstance(container_client, FileSystemClient):  # gen2
-            if container_client.get_directory_client(non_wildcard_path).exists():
-                paths = container_client.get_paths(non_wildcard_path, True)
-                file_names = [path.name[len(non_wildcard_path):] for path in paths if not path.is_directory]
+        try:
+            if isinstance(container_client, FileSystemClient):  # gen2
+                if container_client.get_directory_client(non_wildcard_path).exists():
+                    paths = container_client.get_paths(non_wildcard_path, True)
+                    file_names = [path.name[len(non_wildcard_path):] for path in paths if not path.is_directory]
+                else:
+                    return False
+            else:  # blob
+                blobs = container_client.list_blobs(name_starts_with=non_wildcard_path)
+                file_names = [blob.name[len(non_wildcard_path):] for blob in blobs]
+        except HttpResponseError as hre:
```

Comment:
need to check if the credential is AmlOboCredential to raise this UMAI error msg, otherwise, it could be SAS token doesn't have enough permission/expired or incorrect account key/client_id/client_credential, etc.

Reply:
fixed